Cocoa/Plug-Ins/Hex Editor/HexWindowController.mm

```diff
@@ -230,6 +232,10 @@ - (void)refreshData:(NSData *)data;
 	NSMutableParagraphStyle *paragraph = [[NSParagraphStyle defaultParagraphStyle] mutableCopy];
 	[paragraph setLineBreakMode:NSLineBreakByCharWrapping];
 	NSDictionary *dictionary = [NSDictionary dictionaryWithObject:paragraph forKey:NSParagraphStyleAttributeName];
+
+	NSMutableParagraphStyle *hexParagraph = [[NSParagraphStyle defaultParagraphStyle] mutableCopy];
+	//[hexParagraph setLineBreakMode:NSLineBreakByCharWrapping];
```

Comment:
can you remove this line too? if it wraps at all, the next line gets screwed up, so we'd never uncomment it